### PR TITLE
feat(linting): add conventional prettier setup

### PR DESCRIPTION
**Pull request by abeforgit**
same config as frontend GN, which has been working well for us
single quotes in JS, double quotes in hbs files
- feat(linting): add conventional prettier setup
- feat(formatter): add prettier ignore file
- feat(formatter): add rc file for prettier
- fix(formatting): run prettier on all files

### CODE REVIEW

**Review by elpoelma — APPROVED** [review]

File: models/task.js [hunk 1]
```diff
@@ -30,18 +35,40 @@ export default class Task {
      PREFIX    adms: <http://www.w3.org/ns/adms#>
      INSERT DATA {
         ${sparqlEscapeUri(uri)} a task:Task;
-                                                mu:uuid ${sparqlEscapeString(id)};
-                                                adms:status ${sparqlEscapeUri(TASK_STATUS_CREATED)};
-                                                task:numberOfRetries ${sparqlEscapeInt(0)};
-                                                dct:created ${sparqlEscapeDateTime(created)};
-                                                dct:modified ${sparqlEscapeDateTime(created)};
+                                                mu:uuid ${sparqlEscapeString(
+                                                  id
```
Comment (nvdk) [inline models/task.js:39]: I'd argue this is a lot less readable than it was before

File: support/classification-utils.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-const VOORZITTER_VAN_DE_GEMEENTERAAD = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/4c38734d-2cc1-4d33-b792-0bd493ae9fc2';
-const BIJZONDER_COMITE_VOOR_DE_SOCIALE_DIENST = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/5ab0e9b8a3b2ca7c5e000009';
-const DEPUTATIE = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/5ab0e9b8a3b2ca7c5e00000d';
-const GOUVERNEUR = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/180a2fba-6ca9-4766-9b94-82006bb9c709';
-const PROVINCIERAAD = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/5ab0e9b8a3b2ca7c5e00000c';
-const VOORZITTER_VAN_HET_BIJZONDER_COMITE_VOOR_DE_SOCIALE_DIENST = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/53c0d8cd-f3a2-411d-bece-4bd83ae2bbc9';
-const VAST_BUREAU = 'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/5ab0e9b8a3b2ca7c5e000008';
+const BURGEMEESTER =
+  'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/4955bd72cd0e4eb895fdbfab08da0284';
```
Comment (nvdk) [inline support/classification-utils.js:2]: usually a good idea to disable formatting for these type of constant declarations, uri's will always go over the line length but there's little value in having them on their own line

File: models/task.js (hunk 1 above)
Comment (abeforgit) [reply models/task.js]: my bad for running it without looking, prettier in general doesn't touch literal strings, but it seems it got confused

File: support/classification-utils.js (hunk 2 above)
Comment (abeforgit) [reply support/classification-utils.js:2]: well I _can_ read more of the url while having two editors open side-by-side now
but it's true that the chances I need to do so are basically non-existant

I'll disable it for the whole file

**Review by nvdk — APPROVED** [review]

**Pull request merged**